tests/integration-tests/tests/tags/test_tag_propagation.py

```diff
@@ -40,17 +48,45 @@ def test_tag_propagation(pcluster_config_reader, clusters_factory, scheduler, os
     - compute node's root EBS volume (traditional schedulers)
     - shared EBS volume
     """
-    config_file_tags = {"ConfigFileTag": "ConfigFileTagValue"}
-    version_tags = {"parallelcluster:version": get_pcluster_version()}
     cluster_config = pcluster_config_reader()
     cluster = clusters_factory(cluster_config)
+
+    # Uses helper function to perform the tests
+    _tag_propagation_checker_helper(cluster, scheduler, os)
+
+    if scheduler == "slurm":
```

Comment:
I believe batch scheduler also support stop and start, check if we can remove the if condition here

Reply:
Removed!